File: src/LEContagem/app/src/main/java/com/example/lecontagem/HomeActivity.java
package com.example.lecontagem;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.LinearLayout;

import androidx.appcompat.app.AppCompatActivity;

public class HomeActivity extends AppCompatActivity {

    LinearLayout cardCamera;
    LinearLayout cardEquipe;
    LinearLayout cardHistorico;
    LinearLayout cardExportar;
    LinearLayout cardPerfil;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.home);

        cardCamera = findViewById(R.id.cardCamera);
        cardEquipe = findViewById(R.id.cardEquipe);
        cardHistorico = findViewById(R.id.cardHistorico);
        cardExportar = findViewById(R.id.cardExportar);
        cardPerfil = findViewById(R.id.cardPerfil);

        cardCamera.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(HomeActivity.this, ContagemActivity.class);
                startActivity(intent);
            }
        });

        cardEquipe.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(HomeActivity.this, EquipeActivity.class);
                startActivity(intent);
            }
        });

        cardHistorico.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(HomeActivity.this, HistoricoActivity.class);
                startActivity(intent);
            }
        });

        cardExportar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(HomeActivity.this, ExportacaoActivity.class);
                startActivity(intent);
            }
        });

        cardPerfil.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(HomeActivity.this, PerfilActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: src/LEContagem/app/src/main/java/com/example/lecontagem/HomeActivity.kt
package com.example.lecontagem

import android.content.Intent
import android.os.Bundle
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity

class HomeActivity : AppCompatActivity() {

    private lateinit var cardCamera: LinearLayout
    private lateinit var cardEquipe: LinearLayout
    private lateinit var cardHistorico: LinearLayout
    private lateinit var cardExportar: LinearLayout
    private lateinit var cardPerfil: LinearLayout

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.home)

        cardCamera = findViewById(R.id.cardCamera)
        cardEquipe = findViewById(R.id.cardEquipe)
        cardHistorico = findViewById(R.id.cardHistorico)
        cardExportar = findViewById(R.id.cardExportar)
        cardPerfil = findViewById(R.id.cardPerfil)

        cardCamera.setOnClickListener {
            startActivity(Intent(this, ContagemActivity::class.java))
        }

        cardEquipe.setOnClickListener {
            startActivity(Intent(this, EquipeActivity::class.java))
        }

        cardHistorico.setOnClickListener {
            startActivity(Intent(this, HistoricoActivity::class.java))
        }

        cardExportar.setOnClickListener {
            startActivity(Intent(this, ExportacaoActivity::class.java))
        }

        cardPerfil.setOnClickListener {
            startActivity(Intent(this, PerfilActivity::class.java))
        }
    }
}